src/enums/payment-method.ts

```diff
@@ -0,0 +1,13 @@
+export enum PaymentMethod {
+    DIRECT_PAY = 'DIRECT_PAY', // Credit Card
+    INTERNAL = 'INTERNAL', // Staff payment: Routing Slip and No Fee
+    DRAWDOWN = 'DRAWDOWN', // BCOL
```

Comment:
Should this be `DRAW_DOWN` maybe? Feels like 2 words but i could be wrong :)

Reply:
Thanks for checking but it is DRAWDOWN. At least it is the way it comes from pay-api.